state/memory.go

```diff
@@ -217,12 +223,28 @@ func (db *inMemoryStateDB) SetState(addr common.Address, key common.Hash, value
 	db.state.storage[slot{addr, key}] = value
 }
 
-func (db *inMemoryStateDB) Suicide(addr common.Address) bool {
+func (db *inMemoryStateDB) GetStorageRoot(addr common.Address) common.Hash {
+	panic("GetStorageRoot not implemented")
+}
+
+func (db *inMemoryStateDB) GetTransientState(addr common.Address, key common.Hash) common.Hash {
+	panic("GetTransientState not implemented")
+}
+
+func (db *inMemoryStateDB) SetTransientState(addr common.Address, key common.Hash, value common.Hash) {
+	panic("SetTransientState not implemented")
+}
```

Comment:
I think its time to merge [this](https://github.com/Fantom-foundation/Aida/pull/1103) PR so you can use the implementation here. I wouldnt like to break your branch so you can just merge that branch into this one.

Reply:
No right now. As mentioned in another comment, I have to use an older version of carmen without `TransientState`. This PR https://github.com/Fantom-foundation/Sonic/pull/144 has to be merged first.

Reply:
ic